pandas/tests/groupby/transform/test_numba.py

```diff
@@ -129,20 +129,25 @@ def func_1(values, index):
     tm.assert_frame_equal(expected, result)
 
 
+# TODO: Test more than just reductions (e.g. actually test transformations once we have
 @td.skip_if_no("numba")
 @pytest.mark.parametrize(
     "agg_func", [["min", "max"], "min", {"B": ["min", "max"], "C": "sum"}]
 )
-def test_multifunc_notimplimented(agg_func):
```

Comment:
Had to change the test since multifuncs (list/dicts of funcs) aren't supported anyways in transform for the cython engine as well.